subtests/docker_cli/create/create_tmpfs.py

```diff
@@ -0,0 +1,38 @@
+"""
+Create container with --tmpfs option and inspect it
+"""
+
+from create import create_base
+from dockertest.output import DockerVersion
+from dockertest.dockercmd import DockerCmd
+from dockertest.output import mustpass
+
+
+class create_tmpfs(create_base):
+
+    def initialize(self):
+        super(create_tmpfs, self).initialize()
+        # Minimum docker version 1.10 is required for --tmpfs
+        DockerVersion().require_server("1.10")
+
+    def run_once(self):
+        super(create_tmpfs, self).run_once()
+
+    def postprocess(self):
+        # Inspect container for --tmpfs
+        subargs = ["--format", "'{{.HostConfig.Tmpfs}}'",
+                   self.sub_stuff['name']]
+        dkrcmd = DockerCmd(self, "inspect", subargs)
+        self.sub_stuff['dkrcmd'] = dkrcmd
+        self.sub_stuff['cmdresult'] = mustpass(dkrcmd.execute())
+        tmpfs_arg = self.config['tmpfs_arg']
+        with_tmpfs = self.sub_stuff['dkrcmd'].stdout.find(tmpfs_arg) != -1
+        self.failif(not with_tmpfs,
+                    "Container was not created with: --tmpfs %s"
+                    % tmpfs_arg)
+        # Change inspect paramerter to get CID
+        subargs = ["--format", "'{{.Id}}'", self.sub_stuff['name']]
+        dkrcmd = DockerCmd(self, "inspect", subargs)
+        self.sub_stuff['dkrcmd'] = dkrcmd
+        self.sub_stuff['cmdresult'] = mustpass(dkrcmd.execute())
+        super(create_tmpfs, self).postprocess()
```

Comment:
Nearly always (in a destructor is virtually the only exception) methods call the super-class method first.  This would be a great place to add a comment, explaining why it's done last (I can guess, but better to have it in a comment).